test/gtest/ucp/ucp_test.h

```diff
@@ -297,6 +297,15 @@ class ucp_test : public ucp_test_base,
         }
     }
 
+    template <typename T>
+    void wait_for_value(volatile T *var, T value, double timeout = 10.0) const
```

Comment:
maybe better have single version of `wait_for_value` and others rather than having them in UCT and UCP?

Reply:
there are different interfaces for progress - we will need deep refactoring to unite it, definitely not in this release

Reply:
ok

Reply:
could add templated function which accepts 'class Test' which is expected to have 'short_progress_loop' trait
but not in this PR